App/Containers/LaunchScreen.js

```diff
@@ -21,9 +45,7 @@ export default class LaunchScreen extends Component {
               Devtalk vote app!
             </Text>
           </View>
-          <View style={styles.section}>
-            <Button title="Admin Panel" color={colors.green} onPress={() => navigation.navigate('AdminScreen')} />
-          </View>
+          <AdminPanelButton navigation={navigation} uid={userData.uid} adminList={adminList} />
```

Comment:
Why would you give a button a list of all admins in an app just to select one of them?
Why would a button need to know about roles?
Just make a selector `selectIsAdmin` and based on this value render button or not